apps/web/components/Charts/v2/types/index.ts

```diff
@@ -52,3 +61,33 @@ export interface ComponentStyle {
   renderIndex: number
   renderIndexForType: number
 }
+
+export interface CustomStyleConfig {
```

Comment:
How is the documentation of this object in Contentful for developers/Content editors of the supported structure? I see the field in Contentful is a free form JSON?

Reply:
Yeah I think I added (or meant to add) a "Please consult with a developer" hint below.

For now that's how it is. But this uses the same terms as the [Recharts API](https://recharts.org/en-US/api).